users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+
+
+class CustomUser(AbstractUser):
+    phone = models.CharField(max_length=15, blank=True)
```

Comment:
This field allows all sorts of non-phone number inputs which could cause problems in the application. Can you use a field with stricter validation? There may be som Django libraries that implement a phone number field.

Reply:
@adamstep  sure, among all other libraries,  this one looks like an active project and also has `PhoneNumberField ` field at serializer level https://github.com/stefanfoulis/django-phonenumber-field, updated with the PR with a phone number field.